script/Ciel/Day 8/08_01_CIEL08_1C.txt

```diff
@@ -262,22 +276,27 @@ However...
 [sha:3372298c0e26b47101cb613b38d651b5be82684a]{
 -- Page 88, Offset 18473.
 -- 「おまえ、それ言っていいの俺じゃないか、普通!?」
-"Hey, I should be the one saying that, no?!"
+"Hey, shouldn't I should be the one saying that?!"
```

Comment:
This sentence is a bit unclear like this. Could use a context verb.

> Hey, shouldn't I be the only one who can strike that kinda deal?!

Or something in that vein.